### PR TITLE
Add back link variation guidance

### PR DESCRIPTION
**Pull request by danbrady**
@humancompanion-usds Please validate I captured the guidance correctly. Thanks!

### CODE REVIEW

File: src/_components/link/index.md [hunk 1]
```diff
@@ -60,6 +64,16 @@ anchors:
 * **Collections, such as Hub pages.** Active links can be seen on [Hub pages]({{ site.baseurl }}/templates/hub#example)
 * **Less prominent links.** For links that need less prominence than an [Action link]({{ site.baseurl }}/components/link/action) and may appear in a [collection]({{ site.baseurl }}/components/link/collection), we recommend using an Active Link. Active Links have a hover behavior that includes a background color change and an animated right-facing chevron icon for more emphasis. 
 
+### When to use a Back link
+* **As an alternate to the breadcrumb for specific form steps.** The following conditions must be met in order to implement the back link variation:
+  1. The page template must use the [minimal header]({{ site.baseurl }}/components/header/header-minimal) and the [minimal footer]({{ site.baseurl }}/components/footer/footer-minimal).
+  2. The breadcrumb navigation should be removed, allowing the back link to replace it within the layout.
+  3. The form step must follow the [one thing per page pattern]({{ site.baseurl }}/patterns/ask-users-for/a-single-response)
```
Comment (Andrew565) [inline src/_components/link/index.md:71]: Forgot to include a period at the end of the sentence.

**Review by Andrew565 — APPROVED** [review]

File: src/_components/link/index.md [hunk 2]
```diff
@@ -60,6 +64,16 @@ anchors:
 * **Collections, such as Hub pages.** Active links can be seen on [Hub pages]({{ site.baseurl }}/templates/hub#example)
 * **Less prominent links.** For links that need less prominence than an [Action link]({{ site.baseurl }}/components/link/action) and may appear in a [collection]({{ site.baseurl }}/components/link/collection), we recommend using an Active Link. Active Links have a hover behavior that includes a background color change and an animated right-facing chevron icon for more emphasis. 
 
+### When to use a Back link
+* **As an alternate to the breadcrumb for specific form steps.** The following conditions must be met in order to implement the back link variation:
+  1. The page template must use the [minimal header]({{ site.baseurl }}/components/header/header-minimal) and the [minimal footer]({{ site.baseurl }}/components/footer/footer-minimal).
```
Comment (humancompanion-usds) [inline src/_components/link/index.md:69]: You only have to use the minimal header in a conventional form flow (i.e. you've got a Progress bar - Segmented or in other words, this is an online form).

File: src/_components/link/index.md [hunk 3]
```diff
@@ -60,6 +64,16 @@ anchors:
 * **Collections, such as Hub pages.** Active links can be seen on [Hub pages]({{ site.baseurl }}/templates/hub#example)
 * **Less prominent links.** For links that need less prominence than an [Action link]({{ site.baseurl }}/components/link/action) and may appear in a [collection]({{ site.baseurl }}/components/link/collection), we recommend using an Active Link. Active Links have a hover behavior that includes a background color change and an animated right-facing chevron icon for more emphasis. 
 
+### When to use a Back link
```
Comment (humancompanion-usds) [inline src/_components/link/index.md:67]: These are all accurate for a conventional form flow, which is our most common use case. But there are other, I don't know, "mini forms" outside of the Forms Library that have only a step or two that may want to use this (and can). Ask @jeana-adhoc or @mnorthuis for examples.

Comment (jeana-adhoc) [reply src/_components/link/index.md:67]: Here are two things I think of that fall outside of the forms library....

- The [PACT Act Wizard](https://staging.va.gov/pact-act-eligibility/introduction) 
- [Check-in minimal header design](https://www.figma.com/design/sK2vIx6giAynNqUFR0l4Wx/Production-%7C-PCI?node-id=2835-3112&node-type=FRAME&t=JmRP30gX2OI9Ftio-11)

**Comment by danbrady** [comment]
@humancompanion-usds Tweaked the guidance based on our conversation. Let me know if I correctly captured everything. Thanks!

File: src/_components/link/index.md [hunk 4]
```diff
@@ -60,6 +64,16 @@ anchors:
...
+* **As a replacement for breadcrumb** on:
+  * <strong>Conventional Multi-step Forms</strong> that also:
+    * Have a [minimal header]({{ site.baseurl }}/components/header/header-minimal) and [minimal footer]({{ site.baseurl }}/components/footer/footer-minimal)
+    * Follow the [one thing per page pattern]({{ site.baseurl }}/patterns/ask-users-for/a-single-response) pattern
+    * Use the `H1` element represent the headline for the current form page, rather than the step title in the step indicator
```
Comment (jeana-adhoc) [inline src/_components/link/index.md:72]: "to represent" ?

**Review by humancompanion-usds — APPROVED** [review]

**Pull request merged**